.github/actions/sync/shared-config.rb

```diff
@@ -44,14 +44,14 @@ def git(*args)
   homebrew_repository_path/"Library/#{rubocop_yaml}",
   permitted_classes: [Symbol, Regexp],
 )
+homebrew_rubocop_config_yaml["AllCops"]["Exclude"] << '"vendor/**/*"'
```

Comment:
The synced files don't look right (see e.g., https://github.com/Homebrew/rubydoc.brew.sh/pull/651), should this instead be:

```suggestion
homebrew_rubocop_config_yaml["AllCops"]["Exclude"] << "vendor/**/*"
```